### PR TITLE
Feature about

### PR DESCRIPTION
**Pull request by aspencer116**
Add markup and styles for the about section plus the first two products.

### CODE REVIEW

File: src/scss/_svg--animals.scss [hunk 1]
```diff
@@ -57,3 +57,31 @@
   bottom: -1em;
   left: 7.5%;
 }
+
+.lion--element {
+  fill: $green;
+  position: absolute;
+  z-index: -2;
+  width: 18px;
+  height: 15px;
```
Comment (catheraaine) [inline src/scss/_svg--animals.scss:66]: Are these pixels for a reason?

File: src/scss/_typography.scss [hunk 2]
```diff
@@ -19,18 +21,50 @@
...
   text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+.type--link__more {
+  @extend .type--link;
+  text-align: center;
+  margin-top: 1.5em;
+  letter-spacing: 3px;
```
Comment (catheraaine) [inline src/scss/_typography.scss:51]: Same here, you can use fractions of ems, so 0.25em

**Comment by catheraaine** [comment]
Your class names are rockin'

File: src/scss/_svg--animals.scss (hunk 1 above)
Comment (aspencer116) [inline src/scss/_svg--animals.scss:66]: Nope, thanks for catching that

**Pull request merged**